src/connectors/github_organization.py

```diff
@@ -1,35 +1,42 @@
+"""GitHub Webhooks to S3
+Collecting Webhooks from GitHub to S3 using an API Gateway, a Lambda, and Firehose.
```

Comment:
This doesn't seem accurate; the connector brings logs from S3 to Snowflake. There's a separate piece, not in this file, which does the API Gateway/Lambda/Firehose piece to bring the logs into S3.